anaconda_build/enginemeta/meta.yaml

```diff
@@ -5,12 +5,12 @@ requirements:
     run:
```

Comment:
You need to modify the enginemeta version. I would change it to 1.1.0.
Have you used this conda "recipe" to build an enginemeta package locally and then run the following command to create a conda environment you can use to run the stream and ingest engines/handlers successfully?
conda create --use-local -n engine enginemeta ingest_engine stream_engine

Reply:
I have updated the enginemeta version to 1.1.0 and built the enginemeta package locally. I have used that local package to create an engine conda environment and verified that it successfully runs stream_engine, ingest_engine, and the ingest-handlers. In particular, I ran a data ingest and pulled out the recently ingested data via both a sync and an async data request.